src/dtx/dtx_common.c

```diff
@@ -1513,12 +1504,16 @@ start_dtx_reindex_ult(struct ds_cont_child *cont)
 	return rc;
 }
 
-static void
+void
 stop_dtx_reindex_ult(struct ds_cont_child *cont)
 {
 	if (!cont->sc_dtx_reindex || dtx_cont_opened(cont))
 		return;
 
+	/* Do not stop DTX reindex if DTX resync is still in-progress. */
+	if (cont->sc_dtx_resyncing)
+		return;
```

Comment:
Just curious, does it need wait until dtx reindex ULT finish?

Reply:
I do not think so. The DTX reindex ULT holds reference on the container, but does not hold open count. stop_dtx_reindex_ult() is usually called when close the container. It is normal the we still have some references on the container even if the container is closed.